pyproject.toml

```diff
@@ -43,82 +43,54 @@ importlib_metadata = { version = ">=1.0", python  = "<3.8" }  # Granta MI STK re
 ansys-openapi-common = "< 2.0.0"
 ansys-grantami-bomanalytics-openapi = "1.0.0"
 
+[tool.poetry.group.dev.dependencies]
 # Common packages for test, examples, and docs
-jupyterlab = { version = "3.5.0", optional = true }
-lxml = { version = "4.9.1", optional = true }
-pandas = { version = "1.3.5", optional = true }
-tabulate = { version = "0.9.0", optional = true }
-
-# Test and examples packages
-pytest = { version = "7.2.0", optional = true }
-pytest-cov = { version = "4.0.0", optional = true }
-requests-mock = { version = "1.10.0", optional = true }
-ipython = { version = "7.34.0", optional = true }
-
+jupyterlab = { version = "3.5.0" }
+lxml = { version = "4.9.1" }
+pandas = { version = "1.3.5" }
+tabulate = { version = "0.9.0" }
+pytest = { version = "7.2.0" }
+pytest-cov = { version = "4.0.0" }
+requests-mock = { version = "1.10.0" }
+ipython = { version = "7.34.0" }
+
+[tool.poetry.group.doc]
+optional = true
+
+[tool.poetry.group.doc.dependencies]
 # Doc packages
-pyansys-sphinx-theme = { version = "0.3.1", optional = true }
-numpydoc = { version = "1.5.0", optional = true }
-sphinx = { version = "4.5.0", optional = true }
-sphinx_autodoc_typehints = { version = "1.19.1", optional = true }
-sphinx-notfound-page = { version = "0.8.3", optional = true }
-sphinx-copybutton = { version = "0.5.1", optional = true }
-enum_tools = { version = "0.9.0.post1", optional = true }
-sphinx-toolbox = { version = "3.2.0", optional = true }
-jupytext = { version = "1.14.1", optional = true }
-nbsphinx = { version = "0.8.10", optional = true }
-
-[tool.poetry.extras]
-test = [
-  "pytest",
-  "pytest-cov",
-  "requests-mock",
-  "lxml",
-  "tabulate",
-  "pandas",
-  "ipython",
-]
-
-doc = [
-  "pyansys-sphinx-theme",
-  "numpydoc",
-  "sphinx",
-  "sphinx-autodoc-typehints",
-  "sphinx-notfound-page",
-  "sphinx-copybutton",
-  "enum_tools",
-  "sphinx-toolbox",
-  "jupyterlab",
-  "jupytext",
-  "lxml",
-  "tabulate",
-  "pandas",
-  "nbsphinx",
-]
-
-examples = [
```

Comment:
I think we should leave this extra in - it's useful for end users who want to install all the requirements for running the jupyter notebooks.